core/Enums/PackageType.cs

```diff
@@ -76,8 +76,13 @@ public enum PackageType
         Tests = 13,
 
         /// <summary>
-        /// The test
+        /// The Security Portal
+        /// </summary>
+        SecurityPortal = 14,
+
+        /// <summary>
+        /// The specific product database
         /// </summary>
-        SecurityPortal = 14
+        ProductDatabase = 15,
```

Comment:
There was no need, since it is only for DFPackageType, but as discussed the exception for the default PackageTypeHandler getter was improved and a test added.